platform/backend/src/database/seed.ts

```diff
@@ -654,51 +648,3 @@ Provide a brief summary (2-3 sentences) of the key information discovered. Focus
     console.log("✓ Dual LLM configuration already exists, skipping");
   }
 }
-
-/**
- * Seeds MCP catalog from JSON file
- */
-async function seedInternalMcpCatalog(): Promise<void> {
```

Comment:
I don't think we need to "seed" any data here anymore, or at-least once we have the catalog data/installation-flow nailed down